### ISSUE TITLE
Save Makeflow artifacts outside of the container

### ISSUE THREAD

**Issue by Chris-Schnaufer** [AgPipeline/computing-pipeline#189]
**The current behavior or issue**
Unable to leverage the makeflow artifacts preventing detection of "no changes" scenarios and makeflow cleanup.

**The steps taken to reproduce the behavior or issue, or specify a location where the steps were recorded**
Run the workflow. The makeflow artifacts are not available

**Expected behavior**
There's a location where makeflow artifacts can be saved from run to run of container

**Screenshots**
N/A

**Add other supporting information that may be useful**
N/A

**Completion criteria**
Makeflow artifacts can be made available from run to run of container for:
1. detection of previous successful runs (makeflow doesn't cause work to be done)
2. the clean capabilities of makeflow are used by scif 'clean' app

** Technical notes**
* For the "normal" case, another Docker mount at a specific location would allow data persistence. If the expected folder is not available, then it should be created with a warning message; note this results in non-permanent artifact storage.
* For the DE environment, a possible approach is to use a subfolder in the `working` directory

**Comment by Chris-Schnaufer** [comment]
By mounting to the `data` folder associated with the app that is run, makeflow artifacts are automatically saved. For example ```... -v `pwd`/makeflow_artifacts:/scif/data/short_workflow ...```

**Comment by Chris-Schnaufer** [comment]
CyVerse: After conversing with @julianpistorius we've determined that we should make the artifacts available during processing and clean them up upon a successful run. This can be accomplished by specifying the working folder to point to `/scif/data/<app name>`. Specifying a shorter path name doesn't work since the CyVerse mount command replaces the folder and doesn't overlay it

**Comment by Chris-Schnaufer** [comment]
Waiting on PR https://github.com/AgPipeline/drone-makeflow/pull/4 before making a new PR

**Comment by Chris-Schnaufer** [comment]
PR: https://github.com/AgPipeline/drone-makeflow/pull/8
